esphomeRemote/esphomeRemote.h

```diff
@@ -849,6 +909,51 @@ void drawBootSequence() {
   menuDrawing = false;
 }
 
+void drawMenuLightDetail(std::vector<std::shared_ptr<MenuTitleBase>> menuTitles){
+    //
+    //
+    // ##################################################
+    // LEAVE THAT ONE HERE FOR NOW - REMOVE BEFORE MERGE!
+    // IT'S NOW PART OF DRAWMENU!
+    // ##################################################
+    //
+    //
+    // draw button and sliders to adjust light
+    // std::vector<std::shared_ptr<MenuTitleBase>> menuTitles = lightGroup->lights[lightGroup->currentSelectedLight]->lightTitleItems();
+    activeMenuTitleCount = menuTitles.size();
+    if(menuTitles.size() == 0 ) {
+        return;
+    }
+    scrollMenuPosition();
+
+    int menuState = menuIndex;
+    int yPos = id(header_height);
+    int currentSelectedLight = lightGroup->currentSelectedLight;
+    bool lightDetailSelected = lightGroup->lightDetailSelected;
+    // auto brightness_bar =lightGroup->lights[currentSelectedLight]->brightness ;
+    // auto color_temp_bar =0.64 * lightGroup->lights[currentSelectedLight]->color_temp ;
+    for(int i = 0;i<menuTitles.size();++i){
+        auto mt = menuTitles[i];
+        if(mt->titleType == BaseMenuTitleType){
+            ESP_LOGW("WARNING", "add title");
+            drawTitle(menuState, 0,mt->friendlyName, yPos, true);

```

Comment:
i think the xPos should be id(margin_size) here to match the other screens

Reply:
See the comment on function start 😉 
You would be right but this function is no longer used. I developed the sliders there and wanted to keep it in for now but I think I don't need it anymore so I removed it 😄

Reply:
![image](https://user-images.githubusercontent.com/2607659/198498758-2d13897d-7401-43f5-9e02-6b23b60b2865.png)
it was the wrong spot but here's what i meant. can we do these two ui fixes

Reply:
oh sure, I missed that, sorry.